src/test_loader.rs

```diff
@@ -128,7 +140,7 @@ where
 fn get_addresses_from_program<R, Offset>(
     prog: IncompleteLineProgram<R>,
     debug_strs: &DebugStr<R>,
-    entries: &[(u64, LineType)],
+    entries: &Vec<(u64, LineType, &Option<String>)>,
```

Comment:
I had to wrestle with type checker, and therefore this change.
Let me know please if this and other changes bellow don't look ok.